src/client/app/DragonflyPreferencesComponent.jsx

```diff
@@ -59,66 +60,95 @@ class DragonflyPreferencesComponent extends React.Component {
+                <div className="dragon-powered-by divLeft">
+                {
+                  logo  ? <img src={logo} />
+                        : <img src="./images/logo-dragonfly-ii2.png" />
+                }
                 </div>
-                
-                
-                
-                <div className="dragon-select-list-row">
-                    <div className="dragon-select-list-form-cell">
-                        <div className="radio">
-                          <label>
-                            <input className="form-check-input" type="radio" value="text" checked={this.state.selectedContactOption === 'text'} onChange={this.handleContactOptionChange} name="group1" id="radio101"/>
-                            Text to 
-                            
-                          </label>
-                        </div>
+              </a>
+            </div>
+            <form onSubmit={this.handleSubmit}>
+              <h2>Congratulations {first}!</h2>
+              <br/>
+              <h4>Your session is complete and you have earned ${earned}.</h4>
+              <br/>
+              <h4>We are going to pay you through the Venmo App, please indicate your preferred contact method:</h4>
+              <div className="dragon-select-list-row">
+                <div className="dragon-select-list-form-cell">
+                  <div className="radio">
+                    <label>
+                      <input className="form-check-input" type="radio" value="email" checked={this.state.selectedContactOption === 'email'} onChange={this.handleContactOptionChange} name="group1" id="radio100"/>
+                      Email to 
+                    </label>
+                  </div>
+                </div>  
+                <div className="dragon-select-list-form-cell">
+                  <input value={this.state.email} onChange={this.updateEmailValue} className="form-control" placeholder="email address"/>
+                </div>
+              </div>
+              <div className="dragon-select-list-row">
+                <div className="dragon-select-list-form-cell">
+                  <div className="radio">
+                    <label>
+                      <input className="form-check-input" type="radio" value="text" checked={this.state.selectedContactOption === 'text'} onChange={this.handleContactOptionChange} name="group1" id="radio101"/>
+                      Text to 
+                    </label>
+                  </div>
+                </div>
+                <div className="dragon-select-list-form-cell">
+                  <input value={this.state.mobile} onChange={this.updateMobileValue} className="form-control" placeholder="mobile number"/>
+                </div>
+              </div>
+
+              <div className="checkbox">
+                <label><input type="checkbox" onChange={this.showPostAddressForm}/> Add post address</label>
+              </div>
+              { this.state.postAddressForm && (
```

Comment:
Instead of this you can create a function that ruturns null if( !this.state.postAddressForm) and returns jsx component in other case